plaid/src/apis/general/mod.rs

```diff
@@ -12,9 +12,14 @@ use std::time::Duration;
 
 use crate::{data::DelayedMessage, executor::Message};
 
+use super::DEFAULT_TIMEOUT_SECONDS;
+
 #[derive(Deserialize)]
 pub struct GeneralConfig {
     network: network::Config,
+    /// The number of seconds until an external API request times out.
+    /// If `None`, the `DEFAULT_TIMEOUT_SECONDS` will be used.
+    api_timeout_seconds: Option<u64>,
```

Comment:
I wouldn't make this an option, use the serde deserialize with default feature. Then you won't have to unwrap this eveyrwhere.